typings/phone.ts

```diff
@@ -13,7 +13,7 @@ export enum PhoneEvents {
   FETCH_CREDENTIALS = 'npwd:getCredentials',
   ON_INIT = 'npwd:onInit',
```

Comment:
Do we actually use the onInit event anywhere?

Reply:
It's triggered by `handleNewPlayerJoined` for non-multicharacter, and `handleNewPlayerEvent` for multicharacter.

It triggers
```ts
function fetchOnInitialize() {
  emitNet(TwitterEvents.GET_OR_CREATE_PROFILE);
}
```

On that note, the following also triggers that function
```ts
RegisterCommand(
  'phone:restart',
  async () => {
    await hidePhone();
    sendMessage('PHONE', 'phoneRestart', {});
    fetchOnInitialize();
  },
  false,
);
```
but that sendMessage doesn't exist anywhere (I assume it was intended for resetting the UI?)

Reply:
This will most likely be killed with the twitter refactor and is legacy code from an older time. That onInit event is also from that time, which is why I ask.

```js
function fetchOnInitialize() {
  emitNet(TwitterEvents.GET_OR_CREATE_PROFILE);
}
```

Reply:
I prefer if we were to use PLAYER_LOADED and PLAYER_UNLOADED events or a combined SET_PLAYER_LOADED with param. onInit is weird naming from the past.

Reply:
@itschip Since twitter branch is under you, tagging you as this is relevant.